include/ibm/management_console_rest.hpp

```diff
@@ -533,14 +533,32 @@ inline void
             asyncResp->res.jsonValue["TransactionID"] = var;
             return;
         }
-        BMCWEB_LOG_DEBUG << "There is a conflict with the lock table";
-        asyncResp->res.result(boost::beast::http::status::conflict);
+
+        // Check if the session is active. If active, send back the lock
+        // conflict, else clear the stale entry in the lock table.
         auto var =
             std::get<std::pair<uint32_t, LockRequest>>(conflictStatus.second);
+        std::string sessionId = std::get<0>(var.second);
+        auto session =
+            persistent_data::SessionStore::getInstance().getSessionByUid(
+                sessionId);
+
+        if (session == nullptr)
+        {
+            BMCWEB_LOG_DEBUG << "Releasing lock of the session id(stale): "
+                             << sessionId;
+            // clear the stale entry
+            crow::ibm_mc_lock::Lock::getInstance().releaseLock(sessionId);
+            return;
```

Comment:
After releasing this stale lock - before returning the response you should continue processing the lock request - to actually set the lock in the lock table for this current command. 
By calling crow::ibm_mc_lock::Lock::getInstance().acquireLock(t) again, it will create a successful lock record for this new session

Reply:
done